src/components/NavMobile/NavMobile.js

```diff
@@ -0,0 +1,175 @@
+// Dependencies
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+// Components, assets, actions, styles etc..
+import { toggleScroll } from '../../actions/index.js';
+import Button from '../Button/Button';
+import { ReactComponent as HUOALogo } from '../../assets/huoa-logo.svg';
+import { ReactComponent as Hamburger } from '../../assets/hamburger.svg';
+import { ReactComponent as ButtonClose } from '../../assets/button-close.svg';
+import { ReactComponent as Carrot } from '../../assets/carrot.svg';
+import './NavMobile.scss';
+
+const NavMobile = () => {
+  const dispatch = useDispatch();
+  const [ navState, setNavState ] = useState('init');
+  const [ dropdownOpen, setDropdownOpen ] = useState('init');
+  const [ closeDropdown, setCloseDropdown ] = useState('init');
+  const [screenWidth, setScreenWidth] = useState(window.innerWidth)
+  const { routes } = useSelector((state) => state.router);
+  const { title, description } = useSelector((state) => state.settings);
+
+  const resetSideNav = (status) => {
+    setCloseDropdown(status);
+    setDropdownOpen(status);
+    setNavState(status);
+  };
+
+  useEffect(() => {
+    function handleResize() {
+      setScreenWidth(window.innerWidth)
+      if(screenWidth >= 1200){
+        resetSideNav('init');
+      }
+    }
+    window.addEventListener('resize', handleResize)
+  })
+
+  const handleAnimation = (hook) => {
+    switch(hook){
+      case 'init':
+        return '';
+      case false:
+        return '--inactive';
+      case true:
+        return '--active';
+      default:
+        return null;
+    }
+  };
+
+  const handleScroll = () => {
+    if(window.screen.width < 768) dispatch(toggleScroll());
```

Comment:
Is eslint not throwing console errors for these?

```suggestion
    if (window.screen.width < 768) dispatch(toggleScroll());
```